packages/salesforcedx-vscode-core/src/settings/pushOrDeployOnSave.ts

```diff
@@ -128,6 +128,7 @@ export class DeployQueue {
           default:
             displayError(e.message);
         }
+      } finally {
```

Comment:
DeployQueue.locked is set to false. What does this mean?

Reply:
this unlocks the queue in order to let the subsequent pushes on save be triggered